app/pb/api.proto

```diff
@@ -28,6 +28,8 @@ service API {
 
   rpc ListFriends(google.protobuf.Empty) returns (ListFriendsRes);
 
+  rpc ListMutualFriends(ListMutualFriendsReq) returns (ListMutualFriendsRes);
```

Comment:
It should be an error to send in your own user_id here, in my opinion. Either way, it should be documented here what happens when you do so.

Reply:
The way it was handled in models was to return an empty list, so the current method does the same. I imagine it's so that when you look at your own profile it will just have an empty list instead of an error. We can pose the question to the dev team at large or else just leave it as is until it needs changing.

But yes, I'll add a note.

Reply:
I think it is perfectly fine for `org.couchers.api.core.GetUser()` rpc call to return empty list for your own profile, while in this separate rpc call throws an error. Anyway, the actual behaviour is not very important as long as it is documented :)